scripts/check-message-compatibility.py

```diff
@@ -150,8 +150,8 @@ def main(repo1: str, repo2: str, verbose: bool = False):
     else:
         if verbose:
             for msg_type in incompatible_types:
-                file1 = f'{repo1}/msg/{msg_type}.msg'
-                file2 = f'{repo2}/msg/{msg_type}.msg'
+                file1 = os.path.join(repo1, 'msg', f'{msg_type}.msg')
```

Comment:
Just FYI, I noticed the path could end up with a double separator `//` if the user passes a directory with directory syntax `APX4/`. In that case we would get `../APX4//msg/VehicleLocalPosition.msg`.

Python handled it anyways but this is nicer anyways